Sources/Apollo/InterceptorRequestChain.swift

```diff
@@ -25,11 +25,6 @@ final public class InterceptorRequestChain: Cancellable, RequestChain {
   @Atomic public var isCancelled: Bool = false
 
   private var managedSelf: Unmanaged<InterceptorRequestChain>!
-  // This is to fix #2932, caused by overreleasing managedSelf on cancellation of a query or
-  // mutation. It can now only be released once.
-  private lazy var releaseManagedSelf: Void = {
```

Comment:
Removed this singleton 'management' of the `Unmanaged` object so we can retain against on retries.

Reply:
Concerned that this will regress 2932 though. How are we ensuring we don't overrelease?